identity/handler.go

```diff
@@ -164,6 +185,72 @@ func (h *Handler) get(w http.ResponseWriter, r *http.Request, ps httprouter.Para
 	h.r.Writer().Write(w, r, IdentityWithCredentialsMetadataInJSON(*i))
 }
 
+// swagger:route GET /identities/{id}/{identifier} admin getWithCredential
```

Comment:
If the identifier has spaces or slashes or other special characters, this will probably not work well due to encoding issues. Instead, we could add a query parameter which governs if credentials should be included or not in the general identity get API?

Reply:
I almost finish this just couple test to do

currently I put all code the general identity get but I planned to move it to elsewhere if the way is validated

Reply:
done